src/Index.cmake

```diff
@@ -10,34 +10,35 @@ endif()
 
 include_guard()
 
-set(ProjectOptions_SRC_DIR
+set(ProjectOptions_SRC_DIR ${CMAKE_CURRENT_LIST_DIR})
+set(ProjectOptions_SOURCE_DIR
     ${CMAKE_CURRENT_LIST_DIR}
-    CACHE FILEPATH "")
+    PARENT_SCOPE)
```

Comment:
I think these should be removed if they are not used in the module.

Reply:
OK, I wanted to discuss the non common name. 
see [project_options_SOURCE_DIR](https://github.com/cpm-cmake/CPM.cmake#usage)